rest/permissions.py

```diff
@@ -105,7 +106,7 @@ def has_permission(self, request, view):
         except PermissionDenied:
             return False
 
-        if request.method in SAFE_METHODS:
+        if request.method in self.SAFE_METHODS_CUSTOM:
```

Comment:
@giocalitri do i need to create ```edit taxonomy permission``` or it is already present? Can you guide me on this?
Currently i am using this view https://github.com/mitodl/lore/blob/master/rest/views.py#L153 for editing vocabulary. If I create a new ```edit taxonomy permission``` should i also create new API (rest view) that only serves for delete and update.

As you can see this API https://github.com/mitodl/lore/blob/master/rest/views.py#L153 extends ```RetrieveUpdateDestroyAPIView``` which do delete, update, path So should I only need to update permission of this API (see current permission https://github.com/mitodl/lore/blob/master/rest/views.py#L160)